identity-enabler/holder-mobile-app/src/pages/CreatePresentation.svelte

```diff
@@ -47,32 +44,19 @@
         loadingScreen.set();
     });
 
-    function goBack() {
-        if ($modalStatus.active) {
-            modalStatus.set({ active: false, type: null });
-            return;
-        }
-
-        window.history.back();
-    }
-
     function onClickDev() {
         navigate("/tutorial");
     }
 
     function showJSON() {
-        modalStatus.set({
-            active: true,
-            type: "code",
-            props: { code: presentationJSON }
-        });
+        navigate("/code", { state: { code: presentationJSON } });
     }
```

Comment:
same comment

Reply:
Performance issues will get addressed in future PR https://github.com/ZebraDevs/Zebra-Iota-Edge-SDK/pull/76